tembo-operator/src/app_service/ingress.rs

```diff
@@ -54,13 +58,44 @@ fn generate_ingress(
             ..ObjectMeta::default()
         },
         spec: IngressRouteSpec {
-            entry_points: Some(vec!["websecure".to_string()]),
+            entry_points: Some(entry_points),
             routes,
             tls: Some(IngressRouteTls::default()),
         },
     }
 }
 
+fn generate_ingress_tcp(
+    coredb_name: &str,
+    namespace: &str,
+    oref: OwnerReference,
+    routes: Vec<IngressRouteTCPRoutes>,
+    entry_points: Vec<String>,
+) -> IngressRouteTCP {
+    let mut labels: BTreeMap<String, String> = BTreeMap::new();
+    labels.insert("component".to_owned(), COMPONENT_NAME.to_string());
+    labels.insert("coredb.io/name".to_owned(), coredb_name.to_string());
+
+    IngressRouteTCP {
+        metadata: ObjectMeta {
+            // using coredb name, since we'll have 1x ingress per coredb
+            name: Some(coredb_name.to_owned()),
```

Comment:
can you add a short suffix like "-apps" because for old servers, this name is already used for the ingressroute tcp that ingress postgres

Reply:
I was thinking that as well and added the label logic to ignore them in the postgres ingress reconciliation. I'm up for adding this as well though.

Reply:
I had tried this but ran into an issue so punted on it. @EvanHStanton can you try this out and see what happens?

Reply:
Updated